### PR TITLE
feat:create api gateway

### PR DESCRIPTION
**Pull request by MeloLawson**
api gateway creation with eureka server to manage the microservices

linked #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +23,7 @@ POSTGRES_USER=*******
 
 Be careful ! If you're not using the last command, but the running tool of your IDE, uncomment the second line in the application.properties of the corresponding module.
 
+### Use of API-GATEWAY and EUREKA-SERVER
+To manage the microservices of the application, you need to first launch the module eureka-server (see the command above).
+Then launch the module api_gateway.
```
Comment (MelvinCou) [inline README.md:28]: `eureka-server` and `api_gateway` choose `_` or `-`.

**Review by MelvinCou — CHANGES_REQUESTED** [review]
Works great!
Just fix documentation.

File: README.md [hunk 2]
```diff
@@ -21,3 +23,7 @@ POSTGRES_USER=*******
 
 Be careful ! If you're not using the last command, but the running tool of your IDE, uncomment the second line in the application.properties of the corresponding module.
 
+### Use of API-GATEWAY and EUREKA-SERVER
+To manage the microservices of the application, you need to first launch the module eureka-server (see the command above).
```
Comment (PaulIRVOAS) [inline README.md:27]:
```suggestion
To manage the microservices of the application, you need to first launch the module eureka_server (see the command above).
```
eureka-server not exist, please change to eureka_server

**Review by PaulIRVOAS — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ Original repository: https://github.com/MelvinCou/cash-manager
 ## Initialization of the back project (CashManagerServer)
 This part of the project is organized as modules, that can be launch independently
 
+### Launch one module
+
 - Go the folder CashManagerServer with the command cd CashManagerServer
```
[the remark this answers is not in the document]
Comment (MeloLawson) [reply README.md]: done !

**Review by MelvinCou — APPROVED** [review]

**Pull request merged**